.github/workflows/rtw-tests.yml

```diff
@@ -78,3 +78,8 @@ jobs:
           conda activate rtw
           cd esmvaltool/utils/recipe_test_workflow/doc
           make linkcheck
+
+      - name: Lint shell script with ShellCheck
+        run: |
+          shellcheck esmvaltool/utils/recipe_test_workflow/recipe_test_workflo\
+          w/app/get_esmval/bin/clone_latest_esmval.sh
```

Comment:
Would it be possible to find all `.sh` files, then run `shellcheck`, please?

Reply:
This should now have been added in this file https://github.com/ESMValGroup/ESMValTool/blob/5f016dc3c80001768ecd0134acb3aaa84cc58d22/.github/workflows/check-rtw.yml#L76-L80